.github/workflows/project.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  pull_request_target:
```

Comment:
I was going to ask why `pull_request_target` instead of `pull_request`, but this will make it compatible for service maintainers in the future, so yay! 👍